src/api/cancel_order.js

```diff
@@ -0,0 +1,12 @@
+const { post } = require('request-promise')
+
+module.exports = (client, id) => {
+  const url = client.config.api + '/cancelOrder'
+
+  const data = {
+    orderId: id,
+    signature: 'web3signature'
+  }
```

Comment:
@plutoegg / @prdn do we have more specs for this? 

Or do we simply sign `{ orderId }` ?

Reply:
Just orderId should be fine